sdk/src/lib.rs

```diff
@@ -144,6 +144,10 @@ pub use solana_sdk_macro::declare_deprecated_id;
 /// assert_eq!(id(), my_id);
 /// ```
 pub use solana_sdk_macro::declare_id;
+#[deprecated(
+    since = "2.1.0",
+    note = "Use `solana_program::pubkey::Pubkey::from_str_const` instead"
```

Comment:
Here it should say "Use `solana_pubkey::pubkey` ... ", right?

Reply:
Good spot, I've also updated it to re-export solana_pubkey::pubkey instead of the one from solana_sdk_macro